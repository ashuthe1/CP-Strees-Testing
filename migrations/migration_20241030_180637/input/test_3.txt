10
5 3 4 0 0 -2 6 1 6 -10 
97
22 40
11 24
25 38
49 51
4 38
7 53
16 52
14 22
44 46
10 30
10 31
8 55
9 28
7 39
2 17
15 42
38 38
20 39
21 49
1 18
7 25
16 21
9 29
42 52
3 19
28 49
33 48
22 41
20 29
22 24
25 32
14 50
8 20
21 43
8 41
6 10
14 21
9 17
36 45
48 55
41 47
14 25
33 36
39 52
8 51
14 47
32 41
26 27
34 48
14 33
6 41
29 41
26 45
6 37
29 34
12 47
6 14
36 46
20 28
10 16
8 27
23 45
35 36
16 27
2 16
4 29
19 21
2 33
38 50
28 38
11 17
24 40
7 9
1 36
16 26
32 39
10 41
16 45
8 31
9 16
7 33
25 40
25 26
17 20
8 41
18 34
2 5
10 28
28 54
11 32
30 50
37 40
22 52
16 30
14 18
25 46
24 49