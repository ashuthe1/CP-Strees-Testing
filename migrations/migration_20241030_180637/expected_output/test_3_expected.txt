81
100
55
11
258
295
191
73
18
140
144
265
165
235
175
145
5
88
133
198
178
57
165
55
193
86
76
79
51
14
36
199
135
92
218
79
69
88
38
13
32
92
10
65
276
182
34
20
65
130
244
36
77
229
19
196
103
44
51
53
182
96
-2
100
163
232
16
270
66
26
52
75
56
274
95
35
178
157
184
76
216
67
24
42
218
96
44
142
93
136
87
21
134
98
53
92
123
